### DISCUSSION TITLE
i´m using docker it should upload from [Jan 20]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4954] [Discord Threads]
> 🙋‍♂️ **Soldado**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Soldado**  
#### i changed from viewer to creator in permissions  
#  
> 🙋‍♂️ **Soldado**  
#### in second line it says permission not allowed  
![image0](https://cdn.discordapp.com/attachments/1054755000011722793/1054784874436907150/Screenshot_2022-12-20_at_15.36.45.png)  
#  
> 🙋‍♂️ **Soldado**  
#### more updates:   
#### - in viewer mode it´s not possible to upload images (make sense)   
#### - So i changed to creator. It uploads files with no problem   
#### - But when user (creator) logout and after login it assume viewer permissions  
#  
> 💁 **o1lab**  
#### pls raise an issue with all the details  
#  
> 🙋‍♂️ **Soldado**  
#### ok, done in github https://github.com/nocodb/nocodb/issues/4684  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1054755000011722793)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
